apps/ledger-live-mobile/e2e/specs/languageChange.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { device, expect } from "detox";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import SettingsPage from "../models/settings/settingsPage";
+import GeneralSettingsPage from "../models/settings/generalSettingsPage";
+import { loadConfig } from "../bridge/server";
+import { delay } from "../helpers";
```

Comment:
I think you can remove this

Reply:
i think i can't

Reply:
It's not being used anywhere in this file though. Try removing it and rerunning the test